apps/frontend/src/app/components/delete-user-dialog/delete-user-dialog.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+// import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+// import { DeleteUserDialogComponent } from './delete-user-dialog.component';
+// import { ThemeModule } from '../../theme.module';
+// import { NbDialogService } from '@nebular/theme';
+
+// describe('ConfirmDialogComponent', () => {
+//   let component: DeleteUserDialogComponent;
+//   let fixture: ComponentFixture<DeleteUserDialogComponent>;
+
+//   beforeEach(async(() => {
+//     TestBed.configureTestingModule({
+//       imports: [ThemeModule],
+//       declarations: [DeleteUserDialogComponent],
+//       providers: [NbDialogService]
+//     }).compileComponents();
+//   }));
+
+//   beforeEach(() => {
+//     fixture = TestBed.createComponent(DeleteUserDialogComponent);
+//     component = fixture.componentInstance;
+//     fixture.detectChanges();
+//   });
+
+//   it('should create', () => {
+//     expect(component).toBeTruthy();
+//   });
+// });
```

Comment:
Why comment this out? I have a branch in review that gets frontend unit tests working, and this looks fine. If you're having any issues with `NbDialogRef`, override it with `[{ provide: NbDialogRef, useValue: {} }]`. See my `refactor/frontend-auth-and-unit-tests` branch for examples of how to unit test Nebular dialogs.

Reply:
Spec was commented out on confirm dialog component, so when I copied it I leff it as is, just changed class names. I will write spec then